Ali GOK browsers week1

aligoknl opened the pull request.

cgduncan7 commented on 2-Browsers/Week1/homework/ex1-bookList/index.js:
```diff
@@ -17,8 +17,33 @@ https: //hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
+    list.appendChild(titleAuthor);
+    list.appendChild(img);
+    
+    book.alreadyRead ? list.style.backgroundColor = 'green' : list.style.backgroundColor = 'red';
```
Fancy! However, ternaries are usually written like

```suggestion
    list.style.backgroundColor = book.alreadyRead ? 'green' : 'red';
```

cgduncan7 commented on 2-Browsers/Week1/homework/ex2-aboutMe/index.js:
```diff
@@ -8,5 +8,17 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 3. Look in the css file!
 ------------------------------------------------------------------------------*/
+const nicknameEl = document.getElementById("nickname");
+nicknameEl.textContent = "Crazzy Bob";
```
Really?  😆

cgduncan7 commented on 2-Browsers/Week1/homework/ex4-whatsTheTime/index.html:
```diff
@@ -6,6 +6,7 @@
     <title>What's The Time?</title>
   </head>
   <body>
+    <div id = "clock" onload="addCurrentTime()"></div>
```
Nice! I think we were aiming to get you to use `window.addEventListener('load', addCurrentTime)` but this does the trick as well 😄

cgduncan7 commented on 2-Browsers/Week1/homework/ex4-whatsTheTime/index.js:
```diff
@@ -8,7 +8,24 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
+    
+  let time = hours + ":" + minutes + ":" + seconds;
+  document.getElementById("clock").textContent = time; 
+  setTimeout(function(){addCurrentTime()}, 1000);
```
If you want to pass a function as an argument to another function you can just write the function but without the ending params. Don't need to wrap it in an anonymous function

```suggestion
  setTimeout(addCurrentTime, 1000);
```

cgduncan7 commented on 2-Browsers/Week1/homework/ex5-catWalk/index.js:
```diff
@@ -21,8 +21,27 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
    https://media1.tenor.com/images/2de63e950fb254920054f9bd081e8157/tenor.gif
 -----------------------------------------------------------------------------*/
+const cat = document.querySelector('img');
+let leftHand = 0;
```
Looks good! Maybe use some more clear variable names to increase readability

cgduncan7 commented on 2-Browsers/Week1/homework/ex2-aboutMe/index.js:
```diff
@@ -8,5 +8,17 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
+
+const newClass = document.querySelectorAll("li");
+
+for (let i in newClass){
```
Try not to use `for in`, some weird stuff can happen!

```suggestion
newClass.forEach((li) => {
```

cgduncan7 reviewed (approved):
Great work @aligoknl 👏 I've added a few suggestions but nothing major. Nice submission! 🎖️

aligoknl commented:
> Great work @aligoknl 👏 I've added a few suggestions but nothing major. Nice submission! 🎖️

Thank you very much @cgduncan7 for your valuable time and useful comments. I have just updated the code and made minor changes according to your suggestions.
